mappings/net/minecraft/sortme/MobSpawnerLogic.mapping

```diff
@@ -11,9 +11,14 @@ CLASS bgd net/minecraft/sortme/MobSpawnerLogic
 	FIELD l requiredPlayerRange I
 	FIELD m spawnRange I
 	METHOD a getWorld ()Lbgv;
+	METHOD a spawnEntity (Lahz;)V
+	METHOD a setEntityID (Laid;)V
 	METHOD a setSpawnEntry (Lbhj;)V
 	METHOD a deserialize (Lib;)V
 	METHOD b getPos ()Lev;
 	METHOD b serialize (Lib;)Lib;
 	METHOD c update ()V
 	METHOD d getRenderedEntity ()Lahz;
+	METHOD g getEntityID ()Lqr;
+	METHOD h playerInRange ()Z
```

Comment:
`isPlayerInRange`. Again, names like this `playerInRange` is for field names while verb phrases are for method names.